bundle/language_fr.properties

```diff
@@ -1174,6 +1174,8 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = Parents vivant ensemble
 angal.patient.patientbrowser.title                                                                     = Liste Patients
 angal.patient.patientphoto                                                                             = Photo du Patient
+angal.patient.patientphoto.error.title                                                                 = Erreur de photo du patient
+angal.patient.patientphoto.error.filenotfound                                                          = La photo du patient n'a pas �t� trouv�e.
```

Comment:
There's no need to add new keys in the language other than "en". Please revert this change.